resize.js

```diff
@@ -147,6 +147,12 @@ image.get = function (req, res) {
     return;
   }
 
+  if (params.resolutionX < 0 || params.resolutionY < 0 || params.resolutionX > config.get('constraints.max_width') || params.resolutionY > config.get('constraints.max_height')) {
+    res.writeHead(400, 'Bad request');
+    res.end('The requested image size falls outside of the allowed boundaries of this service.');
```

Comment:
A bit mustard after the meal, but this isn't easily processable by clients. I would suggest to use status code 413 (payload too large) instead, and put some JSON error message in the body.